addons/kommander/1.173.x/kommander-1.yaml

```diff
@@ -45,11 +45,7 @@ spec:
         konvoy:
           allowUnofficialReleases: true
         kubeaddonsRepository:
```

Comment:
You can safely remove this whole section actually and just stick with the chart defaults.